src/core/cgroup.c

```diff
@@ -3041,7 +3041,7 @@ int unit_check_oomd_kill(Unit *u) {
         else if (r == 0)
                 return 0;
 
-        r = cg_get_xattr_malloc(SYSTEMD_CGROUP_CONTROLLER, u->cgroup_path, "user.oomd_kill", &value);
+        r = cg_get_xattr_malloc(SYSTEMD_CGROUP_CONTROLLER, u->cgroup_path, "user.oomd_ooms", &value);
```

Comment:
Hmm, so is this something that is a public API? Is the change in the name something that should be mentioned in NEWS and treated as a backwards-incompatible change? @anitazha

Reply:
> Hmm, so is this something that is a public API?

The oomd xattrs weren't intended to be public API and isn't documented. So I don't think this is backward incompatible change unless for some reason pid1 and systemd-oomd are on different versions.